src/translations/fr.json

```diff
@@ -797,6 +797,10 @@
     "ShuntCompensatorErrorCurrentLessThanZero": "Le nombre de sections utilisées doit être supérieur ou égal à 0",
     "ShuntCompensatorErrorCurrentLessThanMaximum": "Le nombre de sections utilisées doit être inférieur ou égal au nombre maximum",
     "ShuntCompensatorErrorQAtNominalVoltageLessThanZero": "La valeur de Q à Unom doit être supérieure ou égale à 0",
+    "StaticVarCompensatorErrorQFixGreaterThanQMax": "La valeur de Q Fixe doit être inférieure à Q maximum",
```

Comment:
it a little bit weird, maybe : 
- la valeur de Q Fixe doit être inférieure celle de Q maximum, ou à la valeur de Qmaximun
or
- Q Fixe doit être inférieure à Q maximum